<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>618</width>
    <height>443</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="ButtonOpen">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>打开文件</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxq">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>70</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>网格细化</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxV">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>100</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>四面体半径比</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="SpinBoxV">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>95</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <property name="value">
     <double>1.200000000000000</double>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxa">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>130</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>最大体积</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="SpinBoxa">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>127</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="value">
     <double>0.100000000000000</double>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonOut">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>170</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>生成</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>321</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>D:/C/calgrid/pmdc</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelImg">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>210</y>
      <width>300</width>
      <height>200</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonRam">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>200</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>产生点</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonCal">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>240</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>生成</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>618</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
